### DISCUSSION TITLE
from TTS.api not accessible

### DISCUSSION THREAD

**Discussion by PrincessPotion** [idiap/coqui-ai-TTS#62] [Q&A] [answered]
Hi, I'm trying to use this through Python scripting, and after installing following the installation instructions, using "from TTS.api import TTS" gives me an error with the TTS.api stating that it is not accessible. What do?

**Comment by eginhard** [comment, marked as answer]
@PrincessPotion You need to share exactly how you installed the package and how you are running it in order to get help. But probably you didn't install it correctly or are not in the right environment. Can you also share the output of `pip list`?
